### ISSUE TITLE
Failed to create "Role to ExtCell" link.

### ISSUE THREAD

**Issue by SawamiWataru** [personium/personium-client-java#45]
e.g.
```
var as = _p.as('client');
var cell = as.cell();
var role = cell.ctl.role.retrieve({ 'Name': 'testapp-read', '_Box.Name': 'test-app' });
var extCell = cell.ctl.extCell.retrieve('http://localhost/testcell-002/');
role.extCell.link(extCell);  <--- Failed
```

**Comment by SawamiWataru** [comment]
URL Decode is necessary before body setting of linkAPI.
https://github.com/personium/personium-client-java/blob/master/src/main/java/io/personium/client/LinkManager.java#L78

**Comment by SawamiWataru** [comment]
Released v1.4.12
